lib/widgets/map_setting_button.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:trufi_core/blocs/app_review_cubit.dart';
+import 'package:trufi_core/blocs/home_page_cubit.dart';
+
+import 'package:trufi_core/blocs/preferences_cubit.dart';
+import 'package:trufi_core/plan/setting_panel/setting_panel.dart';
+import 'package:trufi_core/plan/setting_panel/setting_panel_cubit.dart';
+import 'fetch_error_handler.dart';
+
+class MapSettingButton extends StatelessWidget {
+  const MapSettingButton({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton(
+      mini: true,
+      backgroundColor: Theme.of(context).backgroundColor,
+      onPressed: () async {
+        final oldSettingPanelCubit = context.read<SettingPanelCubit>().state.copyWith();
```

Comment:
Why do you call here `copyWith`?

Reply:
Before it was necessary but now not, I forgot to remove it, thanks :)